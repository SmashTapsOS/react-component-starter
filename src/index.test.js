import React from "react";
import MyComponent from "./index";
import renderer from "react-test-renderer";

it("renders properly", () => {
  const tree = renderer.create(<MyComponent />).toJSON();
  expect(tree).toMatchSnapshot();
});
