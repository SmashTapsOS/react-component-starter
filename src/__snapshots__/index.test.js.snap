// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<div>
  Hello there
</div>
`;
